File: jats-js/src/test/ts/demo/model/Modifier.ts
class Modifier
{
	private modifierEntries : ModifierEntry[];

	private id : number;

	private minPerComponent : number;

	private maxPerComponent : number;

	private minPerModifier : number;

	private maxPerModifier : number;

	private defaultQuantity : number;

	private defaultIncluded : boolean;

	public getModifierEntries() { return this.modifierEntries; }
	public setModifierEntries( modifierEntries : ModifierEntry[] ) { this.modifierEntries = modifierEntries; }

	public getId() { return this.id; }
	public setId( id : number ) { this.id = id; }

	public getMinPerComponent() { return this.minPerComponent; }
	public setMinPerComponent( minPerComponent : number ) { this.minPerComponent = minPerComponent; }

	public getMaxPerComponent() { return this.maxPerComponent; }
	public setMaxPerComponent( maxPerComponent : number ) { this.maxPerComponent = maxPerComponent; }

	public getMinPerModifier() { return this.minPerModifier; }
	public setMinPerModifier( minPerModifier : number ) { this.minPerModifier = minPerModifier; }

	public getMaxPerModifier() { return this.maxPerModifier; }
	public setMaxPerModifier( maxPerModifier : number ) { this.maxPerModifier = maxPerModifier; }

	public getDefaultQuantity() { return this.defaultQuantity; }
	public setDefaultQuantity( defaultQuantity : number ) { this.defaultQuantity = defaultQuantity; }

	public isDefaultIncluded() { return this.defaultIncluded; }
	public setDefaultIncluded( defaultIncluded : boolean ) { this.defaultIncluded = defaultIncluded; }


}

File: jats-demo-model/src/main/java/com/github/pepe79/jats/demo/model/base/Modifier.java
package com.github.pepe79.jats.demo.model.base;

import java.util.List;

public class Modifier {
	private int id;

	private int minPerModifier;

	private int maxPerModifier;

	private int minPerComponent;

	private int maxPerComponent;

	private int defaultQuantity;

	private boolean defaultIncluded;

	private List<ModifierEntry> modifierEntries;

	public List<ModifierEntry> getModifierEntries() {
		return modifierEntries;
	}

	public void setModifierEntries(List<ModifierEntry> modifierEntries) {
		this.modifierEntries = modifierEntries;
	}

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public int getMinPerComponent() {
		return minPerComponent;
	}

	public void setMinPerComponent(int minPerComponent) {
		this.minPerComponent = minPerComponent;
	}

	public int getMaxPerComponent() {
		return maxPerComponent;
	}

	public void setMaxPerComponent(int maxPerComponent) {
		this.maxPerComponent = maxPerComponent;
	}

	public int getMinPerModifier() {
		return minPerModifier;
	}

	public void setMinPerModifier(int minPerModifier) {
		this.minPerModifier = minPerModifier;
	}

	public int getMaxPerModifier() {
		return maxPerModifier;
	}

	public void setMaxPerModifier(int maxPerModifier) {
		this.maxPerModifier = maxPerModifier;
	}

	public int getDefaultQuantity() {
		return defaultQuantity;
	}

	public void setDefaultQuantity(int defaultQuantity) {
		this.defaultQuantity = defaultQuantity;
	}

	public boolean isDefaultIncluded() {
		return defaultIncluded;
	}

	public void setDefaultIncluded(boolean defaultIncluded) {
		this.defaultIncluded = defaultIncluded;
	}

}
